kernel/src/net/socket/netlink/table/mod.rs

```diff
@@ -113,14 +151,48 @@ impl ProtocolSocketTable {
 
         Ok(BoundHandle::new(self.id, port, addr.groups()))
     }
+
+    fn unicast(&self, dst_port: PortNum, message: AnyUnicastMessage) -> Result<()> {
+        let Some(receiver) = self.unicast_sockets.get(&dst_port) else {
+            // FIXME: Should we return error here?
+            return Ok(());
+        };
+
+        receiver.enqueue_unicast_message(message)
+    }
+
+    fn multicast(&self, message: AnyMulticastMessage) -> Result<()> {
+        let group_id = message.src_addr().groups();
+
+        // FIXME: I guess a message can be sent to only one multiple group.
```

Comment:
- `multiple` -> `multicast`

Also, I don't understand why you want to explicitly check this. This looks very harmless, even if our behavior is inconsistent with the Linux one?